### PR TITLE
README.md: Add basic info on how to add new metrics

### PR DESCRIPTION
**Pull request by LiliC**
and regenerate files.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,4 +66,17 @@ To launch a self contained integration test, run:
...
 ```
 make test-integration
-```
\ No newline at end of file
+```
+
+Adding new metrics to send via telemeter
+-----------
+
+Clone repository locally and in the root of the directory make the following changes:
```
Comment (LiliC) [inline README.md]: Not sure if we want to link the internal docs, or not?

Comment (s-urbaniak) [reply README.md]: I'd say let's add a link here, these docs are gated anyways.

**Comment by s-urbaniak** [comment]
looks good, thanks @LiliC !

**Comment by LiliC** [comment]
@s-urbaniak PTAL, added the link to the doc. Thanks!

**Pull request merged**